core/src/main/java/org/frankframework/util/JdbcUtil.java

```diff
@@ -461,110 +448,26 @@ public static OutputStream getBlobOutputStream(IDbmsSupport dbmsSupport, Object
-					charset = StreamUtil.DEFAULT_INPUT_STREAM_ENCODING;
-				}
-				if (compressBlob) {
-					try (DeflaterOutputStream dos = new DeflaterOutputStream(out)) {
-						dos.write(content.getBytes(charset));
-					}
-				} else {
-					out.write(content.getBytes(charset));
-				}
-			}
-			dbmsSupport.updateBlob(rs, columnIndex, blobHandle);
-		} else {
-			log.warn("content to store in blob was null");
-		}
-	}
-
-	public static void putByteArrayAsBlob(IDbmsSupport dbmsSupport, final ResultSet rs, int columnIndex, byte[] content, boolean compressBlob) throws IOException, DbmsException, SQLException {
-		if (content != null) {
-			Object blobHandle = dbmsSupport.getBlobHandle(rs, columnIndex);
-			try (OutputStream out = dbmsSupport.getBlobOutputStream(rs, columnIndex, blobHandle)) {
-				if (compressBlob) {
-					try (DeflaterOutputStream dos = new DeflaterOutputStream(out)) {
-						dos.write(content);
-					}
-				} else {
-					out.write(content);
-				}
-			}
-			dbmsSupport.updateBlob(rs, columnIndex, blobHandle);
-		} else {
-			log.warn("content to store in blob was null");
-		}
-	}
-
-
-	public static String getClobAsString(final IDbmsSupport dbmsSupport, final ResultSet rs, int columnIndex, boolean xmlEncode) throws IOException, JdbcException, SQLException, DbmsException {
+	public static String getClobAsString(final IDbmsSupport dbmsSupport, final ResultSet rs, int columnIndex, boolean xmlEncode) throws IOException, JdbcException, SQLException {
 		Reader reader = dbmsSupport.getClobReader(rs, columnIndex);
 		if (reader == null) {
 			return null;
 		}
 		return StreamUtil.readerToString(reader, null, xmlEncode);
 	}
 
-	public static String getClobAsString(final IDbmsSupport dbmsSupport, final ResultSet rs, String columnName, boolean xmlEncode) throws IOException, DbmsException, SQLException {
-		Reader reader = dbmsSupport.getClobReader(rs, columnName);
-		if (reader == null) {
-			return null;
-		}
-		return StreamUtil.readerToString(reader, null, xmlEncode);
-	}
-
-	public static void putStringAsClob(IDbmsSupport dbmsSupport, final ResultSet rs, int columnIndex, String content) throws IOException, DbmsException, SQLException {
-		if (content != null) {
-			Object clobHandle = dbmsSupport.getClobHandle(rs, columnIndex);
-			try (Writer writer = dbmsSupport.getClobWriter(rs, columnIndex, clobHandle)) {
-				writer.write(content);
-			}
-			dbmsSupport.updateClob(rs, columnIndex, clobHandle);
-		} else {
-			log.warn("content to store in blob was null");
-		}
-	}
-
 	public static void fullClose(Connection connection, ResultSet rs) {
 		if (rs == null) {
-			log.warn("resultset to close was null");
+			log.warn("resultSet to close was null");
 			close(connection);
 			return;
 		}
 		try {
+			if (!rs.getStatement().isClosed()) {
+				rs.getStatement().close();
+			}
 			if (!rs.isClosed()) {
-				try (Statement statement = rs.getStatement()) {
-					//No Operation, just trying to close the statement!
-				}
+				rs.close();
```

Comment:
These changes are Ok too @tnleeuw ? I changed it a bit: previously only the statement was closed. Now I also close the resultSet.